darts/models/forecasting/forecasting_model.py

```diff
@@ -802,6 +824,51 @@ def retrain_func(
                 logger,
             )
 
+        if fit_kwargs is None:
+            fit_kwargs = dict()
+        if predict_kwargs is None:
+            predict_kwargs = dict()
+
+        # sanity checks of the arguments directly exposed by historical_forecasts
+        if "predict_likelihood_parameters" not in predict_kwargs:
+            predict_kwargs[
+                "predict_likelihood_parameters"
+            ] = predict_likelihood_parameters
+        elif (
+            predict_kwargs["predict_likelihood_parameters"]
+            != predict_likelihood_parameters
+        ):
+            logger.warning(
+                "`predict_likelihood_parameters` was provided with contradictory values, "
+                "retaining the value passed with `predict_kwargs`."
+            )
+        if "num_samples" not in predict_kwargs:
+            predict_kwargs["num_samples"] = num_samples
+        elif predict_kwargs["num_samples"] != num_samples:
+            logger.warning(
+                "`num_samples` was provided with contradictory values, "
+                "retaining the value passed with `predict_kwargs`."
+            )
+
+        # fit/predict_kwargs cannot be used to pass arguments used by historical_forecast logic
+        forbiden_args = ["series", "past_covariates", "future_covariates"]
+        fit_invalid_args = forbiden_args + [
```

Comment:
Do we have to prohibit this?
In theory it would be valid to give a validation series that is further in the future than `series`.

I would leave the responsibility to the user until we implement a dedicated logic for validation series in historical forecasting

Reply:
Sounds good, I was overly cautious